.gitignore

```diff
@@ -3,4 +3,6 @@
 *.o
 dist/
 .stack-work/
-*.DS_Store
\ No newline at end of file
+<<<<<<< HEAD
+*.DS_Store
```

Comment:
I'm fine to have `*.DS_Store` in `.gitignore`. The problem is the `<<<<<<< HEAD` thing.